src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/_help.py

```diff
@@ -539,3 +539,89 @@
           text: |-
             az aks wait -g MyResourceGroup -n MyManagedCluster --updated --interval 60 --timeout 1800
 """
+
+# OpenShift command help
+
+helps['openshift'] = """
+     type: group
+     short-summary: (PREVIEW) Manage Azure OpenShift Services.
+"""
+
+helps['openshift create'] = """
+    type: command
+    short-summary: (PREVIEW) Create a new managed OpenShift cluster.
+    parameters:
+        - name: --compute-vm-size -s
+          type: string
+          short-summary: Size of Virtual Machines to create as OpenShift nodes.
+        - name: --compute-count -c
+          type: int
+          short-summary: Number of nodes in the OpenShift node pool.
+        - name: --fqdn
+          type: string
+          short-summary: FQDN for OpenShift API server loadbalancer internal hostname. For example
+                         myopenshiftcluster.eastus.cloudapp.azure.com
+        - name: --aad-client-app-id
+          type: string
+          short-summary: The ID of an Azure Active Directory client application. If not specified, a new Azure Active Directory client is created.
+        - name: --aad-client-app-secret
+          type: string
+          short-summary: The secret of an Azure Active Directory client application.
+        - name: --aad-tenant-id
+          type: string
+          short-summary: The ID of an Azure Active Directory tenant.
+        - name: --vnet-peer-id
+          type: string
+          short-summary: The ID of a subnet in an existing VNet into which to peer the cluster.
+        - name: --vnet-cidr
+          type: string
+          short-summary: The CIDR used on the VNet into which to deploy the cluster.
+        - name: --subnet-cidr
+          type: string
+          short-summary: The CIDR used on the Subnet into which to deploy the cluster.
+
+
+    examples:
+        - name: Create an OpenShift cluster and auto create an AAD Client
+          text: az openshift create -g MyResourceGroup -n MyManagedCluster --fqdn <FQDN>
+        - name: Create an OpenShift cluster with 5 compute nodes and a custom AAD Client.
+          text: az openshift create -g MyResourceGroup -n MyManagedCluster --fqdn <FQDN>
+                --aad-client-app-id <APP_ID> --aad-client-app-secret <APP_SECRET> --aad-tenant-id <TENANT_ID> --compute-count 5
+        - name: Create an Openshift cluster using a custom vnet
```

Comment:
For placeholders, use `{curly_braces}` instead of `<angle_brackets>`.  The angle brackets don't display correctly in online docs.